docs/images/spec/README.md

```diff
@@ -0,0 +1,15 @@
+# Image Source and Update Policy
+
+The Google slides located at [OpenXLA community
+drive](https://docs.google.com/presentation/d/1vr6x3VtV77NPIwR26uiHEJQb407QtRXMUfIi5CSs8aA/edit?usp=drive_link)
+is considered the source of truth for the SVG images contained within this
+directory. These images were downloaded in SVG format to facilitate their
+integration into this project.
+
+For any community-approved modifications or updates, please adhere to the
+following procedure:
+
+1. Implement the necessary changes directly within the source presentation
```

Comment:
I'm not sure if everyone has edit access to the slides, perhaps we should make this say "Create a copy of the source presentation to propose edits, a maintainer will merge the presentation into the source presentation upon PR merge"

Reply:
Makes sense. Re-phrased as follows per your suggestion. PTAL 

```
1. Create a copy of the source presentation linked above. Implement your proposed edits within this copy.
2. Export the revised image(s) from your edited presentation as SVG files. Submit these SVG files, along with a link to your forked presentation, as part of a pull request for review.
3. Upon approval of your PR, a maintainer will merge the submitted SVG files into this directory. Simultaneously, the maintainer will merge the changes from your forked presentation into the original source presentation.
```